src/test/ui/self/elision/README.md

```diff
@@ -61,15 +61,15 @@ Legends:
 | `struct-async.rs`| ✓ | ✓ | N/A |
 | `alias-async.rs`| ✓ | ✓ | N/A |
 | `assoc-async.rs`| ✓ | ✓ | N/A |
-| `ref-self-async.rs` | X | X | α ⟶ β + γ |
```

Comment:
Nit (as noted on Zulip): The cases that have `α ⟶ β + γ` as opposed to `✓ ⟶ β + γ` should still (and are indeed actually) be `X` in the `Pass?` column since that is how the non-`async fn` tests behave.

Reply:
I think I fixed this @Centril